src/phase2.hpp

```diff
@@ -36,6 +36,20 @@ std::vector<uint64_t> RunPhase2(
     uint32_t const num_buckets,
     uint32_t const log_num_buckets)
 {
+    // memory is split in two halves.
+
+    // The first half is used for read cache of the table we're reading.
+
+    // The second half is used for the sort_manager bucket cache, except for
+    // table 7, where we don't use the sort_manager; then it's used as a write
+    // cache for the table, as we update it.
+
+    // As the last step, we compact table 1. At that point the halfes are also
```

Comment:
`halfes` -> halves or halfs

Reply:
this comment is removed in a later commit, so I think I'll leave it misspelled. If I update this commit I think it would make it harder to review, since changes won't come in as commits on top of what's here